test/document.modified.test.js

```diff
@@ -84,31 +75,27 @@ describe('document modified', function(){
   describe('modified states', function(){
     it('reset after save', function(done){
       var db = start()
-        , B = db.model(modelName, collection)
-        , pending = 2;
+        , B = db.model(modelName, collection);
 
       var b = new B;
 
       b.numbers.push(3);
       b.save(function (err) {
```

Comment:
Original test code had a race condition, this was contributing to some of the flakiness of Travis CI tests.